src/test/java/homework16.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class homework16 extends BaseTest {
+@Test
+    public void registrationNavigation(){
+       WebDriver driver = new ChromeDriver();
+       driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+       String URL =  "https://bbb.testpro.io/";
+       String RegistrationURL = "https://bbb.testpro.io/registration.php";
```

Comment:
For variables, the Java naming convention is to always start with a lowercase letter and then capitalize the first letter of every subsequent word.

For example

String **registrationURL**